app/src/components/submissionPage/SnapSubmissionStyles.ts

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import { MentionsInput } from "react-mentions";
+
+/* A list of styled components for SubmissionTextBox.tsx */
```

Comment:
If these styles are exclusively for `SubmissionTextBox` then a filename like `SubmissionTextBox.styles.ts` would be clearer.

However, if these are more generic (as I think they will be) then the comment would be inaccurate.

Either way, we don't need this comment.